onnxscript/function_libs/torch_lib/ops/core.py

```diff
@@ -4383,6 +4383,11 @@ def aten_index_put(
     See implementation of `torch.onnx.symbolic_opset11.index_put
     <https://github.com/pytorch/pytorch/blob/main/torch/onnx/symbolic_opset11.py#L212>`_.
     """
+    if len(indices) > 1 and any(
+        isinstance(indice, torch.onnx._internal.exporter._tensors.SymbolicTensor)
```

Comment:
check `not isinstance(index, int)` instead as we should not reference the private class.

Reply:
What are the possible types here? Only int and SymbolicTensor? I prefer to keep SymbolicTensor because I know exactly which type the function is supposed to handle.